frontend/apps/desktop/src/app/ui-shared/lib/status-indicator/status-indicator.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-status-indicator',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './status-indicator.component.html',
+  styleUrls: ['./status-indicator.component.less'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class StatusIndicatorComponent implements OnInit, OnChanges {
+  @Input() totalTeams: number = 0;
+  @Input() registeredTeams: number = 0;
+
+  fillPercentage: number = 0;
+  showBackground: boolean = true;
+
+  ngOnInit(): void {
+    this.calculateFill();
+  }
+
+  ngOnChanges(): void {
+    this.calculateFill();
+  }
+
+  calculateFill(): void {
```

Comment:
```suggestion
  private calculateFill(): void {
    if (this.totalTeams === 0) {
      this.fillPercentage = 100;
      this.showBackground = false;

    } else {
      const ratio = Math.min(this.registeredTeams / this.totalTeams, 1);
      this.fillPercentage = ratio * 100;
      this.showBackground = this.fillPercentage < 100;
    }
  }
```

So wird das alles in bisschen klarer und die Maximalgröße beträgt dann 100%.